packages/new-polymath-issuer/src/pages/SecurityTokensDividends/Presenter.tsx

```diff
@@ -197,10 +213,34 @@ export const Presenter: FC<Props> = ({
             )}
           </CardFeatureState>
         </GridRow.Col>
+        <GridRow.Col gridSpan={4}>
+          <FormWrapper<Values>
+            // enableReinitialize
+            initialValues={{
+              searchName,
+            }}
+            // onChange={handleSearchChange}
+            render={({ values }) => (
+              <FormItem name="searchName">
+                <FormItem.Label>Search Text</FormItem.Label>
```

Comment:
You don't have to use Formik for this. It's a single input field, so just use the `BaseInput` directly.